WordPress/Jetpack/Classes/System/JetpackWindowManager.swift

```diff
@@ -39,34 +30,49 @@ class JetpackWindowManager: WindowManager {
         AccountHelper.logOutDefaultWordPressComAccount()
     }
 
-    func importAndShowMigrationContent(_ blog: Blog?, failureCompletion: (() -> ())?) {
+    func importAndShowMigrationContent(_ blog: Blog? = nil) {
+        self.migrationTracker.trackWordPressMigrationEligibility()
+
         DataMigrator().importData() { [weak self] result in
             guard let self else {
                 return
             }
 
             switch result {
             case .success:
-                self.migrationTacker.trackContentImportSucceeded()
+                self.migrationTracker.trackContentImportSucceeded()
                 UserPersistentStoreFactory.instance().isJPContentImportComplete = true
                 NotificationCenter.default.post(name: .WPAccountDefaultWordPressComAccountChanged, object: self)
                 self.showMigrationUIIfNeeded(blog)
                 self.sendMigrationEmail()
             case .failure(let error):
-                self.migrationTacker.trackContentImportFailed(reason: error.localizedDescription)
-                failureCompletion?()
+                self.handleMigrationFailure(error)
+                self.migrationTracker.trackContentImportFailed(reason: error.localizedDescription)
```

Comment:
Same as earlier, would be great to track the failure event before the app navigates to another screen as the order of events may be incorrect.

Reply:
Completed in 524c3716df30aa4249daef40a69ff6a5c8ed0b47. Thanks!